antora-ui-camel/src/css/projects.css

```diff
@@ -2,6 +2,21 @@
   text-align: center;
 }
 
+.projects .camel-project {
+  width: 96%;
+  display: inline-flex;
+  border: 1px solid var(--color-smoke-90);
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 1rem 1rem 2rem 1rem;
+  margin: 1rem;
```

Comment:
I observed from the preview that the alignment is not present below < 1024 and the documentation frame appears shrunk compared to that of project frames. It's best to maintain the same structure, so for below < 1024px width, you should perform the following changes: `width: auto` and `margin: auto`.